fbuapp/Views/PostTableView.h

```diff
@@ -9,18 +9,21 @@
 #import <UIKit/UIKit.h>
 
 NS_ASSUME_NONNULL_BEGIN
-@protocol TableViewDelegate <NSObject>
+@protocol PostTableViewDelegate <NSObject>
 
 - (void) favoritePost: (NSString *)post withUser: (NSString *)user;
 - (void) unFavoritePost: (NSString *)post withUser: (NSString *)user;
+- (void) fetchPosts;
 
 @end
 
 @interface PostTableView : UITableView
 
 - (instancetype)initWithUserId:(NSString *)userId;
 
-@property (nonatomic, weak) id <TableViewDelegate> tableViewDelegate;
+@property (nonatomic, weak) id <PostTableViewDelegate> delegate;
+@property (strong, nonatomic) NSArray * posts;
+//@property (nonatomic) NSArray<Post *> *posts;
```

Comment:
Do you still need this?

Reply:
No. The PostTableView no longer needs to delegate to the HomeViewController because the server interactions are being done directly in HomeViewController and are being passed to the array property of PostTableView.